InOut/CMakeLists.txt

```diff
@@ -9,55 +9,48 @@ option(USE_PORTMIDI "Build the PortMIDI I/O module" ON)
 option(USE_IPMIDI "Build the IPMIDI I/O module" ON)
 option(USE_JACK "Build the jack I/O module and opcodes" ON)
 option(USE_ALSA "Build the ALSA I/O module" ON)
-# option(USE_COREAUDIO "Build the CoreAudio I/O module" ON)
-# option(USE_COREMIDI "Build the CoreMIDI I/O Module" ON)
-# option(USE_AUDIOUNIT "Build the CoreAudio AudioUnit I/O module (requires CoreAudio)" ON)
+option(USE_COREMIDI "Build the CoreMIDI I/O Module" ON)
+option(USE_AUDIOUNIT "Build the CoreAudio AudioUnit I/O module (requires CoreAudio)" ON)
 
 list(APPEND CMAKE_REQUIRED_INCLUDES "/usr/local/include")
 
 # FIND LIBRARIES AND HEADERS
 
 if(USE_PORTAUDIO)
     if (USE_VCPKG)
-        find_package(portaudio CONFIG REQUIRED)
+        # vcpkg includes a config find module
+        find_package(portaudio CONFIG)
```

Comment:
Tbh for the VCPKG build I am just failing if it doesn't find the dependency. It is just a lot easier. There is no extra variables or testing if the target exists or not. The VCPKG build is kind of the CI build so I think this is ok. For the other users using their system packages for dependencies, we can silently disable features if not found.

Reply:
Ok, I added REQUIRED back for the vcpkg branches